datetime/datetime12601.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
+
+const currentDate = new Date();
+
+// 1. Get the current date, print to the console:
+console.log(`Current date: ${currentDate.getDate()}.`);
```

Comment:
date, not day

Reply:
Thank you! I've fixed that. Just have a look at the corrected code: is that what you meant?